testfiles/ReportHeatEmission_RefFSRestaurant.idf

```diff
@@ -0,0 +1,3444 @@
+! RefBldgFullServiceRestaurantNew2004_Chicago.idf
+!
+! This example file is based on Version 1.3_5.0 and was transitioned to Version 6.0 using the transition utility
+!
+! DOE Commercial Reference Building
+! Full Service Restaurant, new construction 90.1-2004
+! Version 1.3_5.0
+! EnergyPlus Version 6.0
+! ASHRAE Standards 90.1-2004 and 62-1999
+!
+! Description:  Single story, Two zone building.
```

Comment:
Mention here that this example file shows the use of the heat emissions report and is based on RefBldgFullServiceRestaurantNew2004_Chicago.idf